docs/kubernetes/kctlr-ingress.rst

```diff
@@ -233,45 +224,93 @@ Kubernetes Secrets
 .. literalinclude:: /kubernetes/config_examples/f5-k8s-ingress-tls-secret.yaml
       :caption: TLS Example
       :linenos:
-      :emphasize-lines: 16-18
 
 :fonticon:`fa fa-download` :download:`f5-k8s-ingress-tls-secret.yaml </kubernetes/config_examples/f5-k8s-ingress-tls-secret.yaml>`
 
-.. _add health monitor to ingress:
 
-Create a BIG-IP Health Monitor for an Ingress
----------------------------------------------
+See Example Ingress Resources
+`````````````````````````````
+
+.. _single service:
 
-#. Add the :code:`virtual-server.f5.com/health` annotation to your Ingress resource.
+Single Service
+~~~~~~~~~~~~~~
 
-   .. literalinclude:: /kubernetes/config_examples/f5-k8s-ingress-health-monitor.yaml
-      :caption: Health Monitor Example
-      :linenos:
-      :emphasize-lines: 9-22
+A :dfn:`Single Service` Ingress creates a BIG-IP virtual server and server pool for a single Kubernetes Service.
+
+.. literalinclude:: /kubernetes/config_examples/f5-k8s-single-ingress.yaml
+   :linenos:
+
+:fonticon:`fa fa-download` :download:`f5-k8s-single-ingress.yaml </kubernetes/config_examples/f5-k8s-single-ingress.yaml>`
+
+.. _simple fanout:
+
+Simple Fanout
+~~~~~~~~~~~~~
+
+A :dfn:`Simple Fanout` Ingress creates a BIG-IP virtual server and pools for a group of Kubernetes Services (one pool per Service).
+
+.. literalinclude:: /kubernetes/config_examples/f5-k8s-ingress-fanout.yaml
+   :linenos:
+
+:fonticon:`fa fa-download` :download:`f5-k8s-ingress-fanout.yaml </kubernetes/config_examples/f5-k8s-ingress-fanout.yaml>`
+
+.. _name-based virtual hosting:
+
+Name-based virtual hosting
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A :dfn:`Name-based virtual hosting` ingress creates the following BIG-IP objects:
+
+- One (1) virtual server with one (1) pool for each Service.
+- Local traffic policies that route requests to specific pools based on host name and path.
+
+.. tip::
+
+   If you don't specify any hosts or paths, the BIG-IP device will proxy traffic for all hosts/paths for the Service specified in the :code:`backend` section of the virtual-server annotation.
```

Comment:
I don't think virtual-server annotation is the right term here. I think it should be Ingress spec, or Ingress definition.